<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Wp3_10</class>
 <widget class="QWizardPage" name="Wp3_10">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>WizardPage</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="8" column="0" colspan="2">
    <widget class="QCheckBox" name="checkBox_62">
     <property name="text">
      <string>Требуются дополнительные исследования для выявления симптомов изменения опорного аппарата зубов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_65">
     <property name="text">
      <string>Атрофия альвеолярного края</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBox_64">
     <property name="text">
      <string>Зубная боль</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QCheckBox" name="checkBox_63">
     <property name="text">
      <string>Эксфолиация зубов</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_22">
     <property name="text">
      <string>Симптомы изменения опорного аппарата зубов</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_69">
     <property name="text">
      <string>Гипертрофия альвеолярного края</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_68">
     <property name="text">
      <string>Нуждаемость в протезировании</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_67">
     <property name="text">
      <string>Потеря зубов</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="QRadioButton" name="radioButton_17">
     <property name="text">
      <string>Нет симптомов (К08.)</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
